tests/testthat/test-vivax.R

```diff
@@ -0,0 +1,26 @@
+test_that('Test falciparum switch produces same', {
+  parameters <- get_parameters()
+  parameters_fal <- get_parameters(parasite = "falciparum")
+  expect_equal(parameters, parameters_fal)
+})
+
+test_that('Test vivax model runs', {
+  vivax_parameters <- get_parameters(parasite = "vivax_tmp")
```

Comment:
See previous comment: not sure what "vivax_tmp" is?

Reply:
See above :) It currently also contains all falciparum parameters required to make the model work without the vivax functionality.